docs/current_docs/integrations/snippets/gitlab-runner-config.yml

```diff
@@ -0,0 +1,34 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dagger-custom-config-toml
+data:
+  config.toml: |
+    concurrent = 10
+    [[runners]]
+      environment = ["HOME=/tmp","FF_GITLAB_REGISTRY_HELPER_IMAGE=1", "_EXPERIMENTAL_DAGGER_RUNNER_HOST"="unix:///var/run/dagger/buildkitd.sock"]
```

Comment:
Is setting `HOME=/tmp` correct here? I genuinely don't know what the expectation is in a GitLab Runners, but this seems odd.

Reply:
@chrira any thoughts on this?

Reply:
We need a writable home dir inside the build container.
The pod that is started by the runner can contain any Image.
As mentioned before, we do not know how the Image is built.
We use this config at Puzzle. I would leave it like this.

Reply:
OK, leaving this as is since you are already using it & it's known to work.